docs/shared-content-source/docs/modules/develop/pages/project-structure.adoc

```diff
@@ -88,7 +88,12 @@ Before delving into that, here we list the runtime plugins:
 `CloudflowSparkPlugin`:: SBT plugin for creating Spark Structured Streaming based streamlets.
 `CloudflowFlinkPlugin`:: SBT plugin for creating Flink-based streamlets.
 
-NOTE: Each runtime plugin uses its own default base image. It's `lightbend/akka-base` for Akka, `lightbend/spark` for Spark and `lightbend/flink` for Flink. To customize the base images, set `cloudflowAkkaBaseImage`, `cloudflowSparkBaseImage`, and `cloudflowFlinkBaseImage` in `settings` of the respective runtime plugins in your application `build.sbt`.
+NOTE: Each runtime plugin uses its own default base image. It's `lightbend/akka-base` for Akka, `lightbend/spark` for Spark and `lightbend/flink` for Flink. It could be the case that you're working against your own artifactory registry and therefore prefer to point to a copy or even a customize version of our base images. To customize the base images, set `cloudflowAkkaBaseImage`, `cloudflowSparkBaseImage`, and `cloudflowFlinkBaseImage` in `settings` of the respective runtime plugins in your application `build.sbt`.
+
+##### Example:
+In an Akka-based streamlet the default base image value is `lightbend/akka-base:${CloudflowBasePlugin.CloudflowVersion}-cloudflow-akka-$AkkaVersion-scala-${CloudflowBasePlugin.ScalaVersion}` which is defined in https://github.com/lightbend/cloudflow/blob/7b02d027ed75eb49cfb80224cfa508575527b38e/core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowAkkaPlugin.scala#L31[here].
+This maybe be instantiated (depending on your Cloudflow, Akka and Scala versions) to `lightbend/akka-base:2.0.7-cloudflow-akka-2.6.8-scala-2.12`. To override this base image in your cloudflow application go to its build.sbt and add new line in its settings such as `cloudflowAkkaBaseImage := myRepositoryUrl/myRepositoryPath:2.0.7-cloudflow-akka-2.6.8-scala-2.12`. https://github.com/lightbend/cloudflow/blob/7b02d027ed75eb49cfb80224cfa508575527b38e/examples/call-record-aggregator/build.sbt#L82[Here] would do.
```

Comment:
```suggestion
This may be set to `lightbend/akka-base:2.0.7-cloudflow-akka-2.6.8-scala-2.12` (depending on your Cloudflow, Akka and Scala versions). To override this base image in your cloudflow application go to its build.sbt and add a new line in its settings such as `cloudflowAkkaBaseImage := myRepositoryUrl/myRepositoryPath:2.0.7-cloudflow-akka-2.6.8-scala-2.12`. https://github.com/lightbend/cloudflow/blob/7b02d027ed75eb49cfb80224cfa508575527b38e/examples/call-record-aggregator/build.sbt#L82[Here] would do.
```

Reply:
Not sure about referring to blobs cause they might get outdated. Can you add a simple snippet instead?

Reply:
You can refer to code in the docs, with an include, here is an example of some other docs that refers to some code, you can add tags to the code to pick a specific piece:
```
[source,json]
----
include::{cloudflow-examples-version}@docsnippets:ROOT:example$sensor-data-scala/src/main/blueprint/blueprint.conf[tag=topic-example]
----
```